Examples/Python/src/Hashing.cpp

```diff
@@ -0,0 +1,58 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2024 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "Acts/Plugins/Hashing/HashingAlgorithmConfig.hpp"
+#include "Acts/Plugins/Hashing/HashingTrainingConfig.hpp"
+#include "Acts/Plugins/Python/Utilities.hpp"
+#include "ActsExamples/TrackFinding/SeedingAlgorithmHashing.hpp"
+
+#include <memory>
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+namespace py = pybind11;
+
+using namespace ActsExamples;
+using namespace Acts;
+
+namespace Acts::Python {
+
+void addHashing(Context& ctx) {
+  auto [m, mex] = ctx.get("main", "examples");
```

Comment:
I'd consider adding a sub module for name spacing. We do this at the Python level for other optional dependencies.

Reply:
Does adding `acts.hashing` and `acts.examples.hashing` sub modules a good way? I saw for other optional dependencies that only a sub module of `acts.examples` is added but shouldn't there be a separation between the `plugin` and the `examples`?